### ISSUE TITLE
Why I use the history_prompt but the result also different?

### ISSUE THREAD

**Issue by DrewdropLife** [suno-ai/bark#168]
I use the history_prompt while set the output_full=True, and than put the same input text , but the result sounds so different. So if the history_prompt only mean use the same speaker? 
And, what should I do if I want the output .wav not change whenever I use the same input text ?

**Comment by gkucsko** [comment]
not sure i understand. `output_full` is used so that you created audio can be used as a history prompt in the next generation.
in general, the same text will produce completely different outputs every time you run the model. sometimes it might sound like a high quality studio recording and sometimes multiple people arguing at a baseball game. using a history prompt biases things a bit, so it can help to achieve some speaker consistency

**Comment by AbdullahMakhdoom** [comment]
@gkucsko If the `temp` is set to 0, would there still be some variability in the output for same text prompt?

**Comment by gkucsko** [comment]
yes the temperature just affects scaling of the probability distribution. so theoretically if the right seed is set etc it will predict the same thing multiple times. however if anything is changed, such as a single word different in the input then the result might be completely different rather than similar.

**Comment by gkucsko** [comment]
closing for inactivity and duplication. probably better as a discussion
